docs/spec/nmt.md

```diff
@@ -167,16 +168,89 @@ This is the same as the queried range.
 - In the event that the queried range is outside the range of the NMT leaves indices, an empty `proof` is returned.
 This empty `proof` consists of an empty set of `nodes` and an empty range, where `start` is `0` and `end` is also `0`.
 
-#### Index-based Merkle Range Proof Verification (for leaves with identical namespace IDs)
+#### Index-based Merkle Range Proof Verification (for leaves with identical namespaces)
 
 Let `leaves` refer to the namespaced messages of the leaves in the queried range `[start, end)`, while `T` represents the root of the tree against which the `proof` is being verified.
 The `proof` can be verified by taking the following steps:
 
-- Verify that all the leaves are namespaced messages with the same namespace ID.
+- Verify that all the leaves are namespaced messages with the same namespace.
 - Compute the tree root `T'` using the leaves and the `proof.nodes`.
 If the computed root `T'` is equal to the expected root `T` of the tree, then the `proof` is valid.
 To compute the tree root `T'`, the [namespaced hash function](#namespaced-hash) should be utilized.
 
+### Short Namespace Absence Proof
```

Comment:
No, there is not. It has been expanded on in this [forum post](https://forum.celestia.org/t/demystifying-namespace-merkle-tree-nmt-exploring-soundness-analysis-and-unveiling-partial-absence-non-inclusion-proofs/998).

Reply:
> Can this eventually be combined as a single proof?

Yes, they can. The reason they are not unified is twofold. Firstly, the current implementation does not support short-proof generation yet. And secondly, the original long version is more conventional and familiar to the readers, so discussing them separately can help to avoid confusion and potential questions.